client/src/client-only-routes/ShowCertification.js

```diff
@@ -103,8 +124,31 @@ class ShowCertification extends Component {
       certTitle,
       completionTime
     } = cert;
+
+    let conditionalDonationMessage = '';
+
+    if (signedInUserName === username && !isDonating && !isDonationRequested) {
```

Comment:
This only displays if you navigate to a cert internally.  If you reload a cert it doesn't appear.  Is that intended?

Reply:
hmm I see, should I fetch user then?

Reply:
Not sure.  I think you need to do something similar to `ShowSettings`:

https://github.com/freeCodeCamp/freeCodeCamp/blob/3304b95213a44525ab2912c2e9987cd49013dc63/client/src/client-only-routes/ShowSettings.js#L167-L170

since `isSignedIn` is false when the page is first loaded.  I don't have much time today, or I'd look into this a bit closer.  Sorry! Should be able to tomorrow.